app/components/classroom/styles/room.css

```diff
@@ -48,7 +48,7 @@
 }
 
 .room_div {
-  white-space: 800px;
+  white-space: normal;
```

Comment:
@hcuffy Are you sure this line is needed then? genuine question, you know how bad my css skills are :D